.github/workflows/build_native_runtime.yml

```diff
@@ -0,0 +1,42 @@
+name: Build Native Runtime
+
+on:
+  push:
+    branches: [ master ]
+
+  pull_request:
+    branches: [ master ]
+
+
+jobs:
+  build_native_runtime:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-18.04]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          submodules: recursive
+
+      - name: Set up JDK 11.0.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11.0.8
+
+      - name: Run CMake
+        run: |
+          mkdir build
+          cd build
+          cmake -B . -S ../nativeruntime/cpp
```

Comment:
Before GitHub Action's M1 device supporting, could we try Apple's universal binary building(https://developer.apple.com/documentation/apple-silicon/building-a-universal-macos-binary)? We can create two building jobs, one for Ubuntu and one for macOS. And we can check matrix.os to ensure every building job only run on its needed os.

Reply:
I briefly looked.into this and the limiting factor may also be libICU :( libICU, at least on homebrew, cannot be compiled into a universal library. I'm not a Mac expert though so I'm not sure if there is a workaround. Everything would be a lot simpler without libICU :/

Reply:
libICU actually brings many problems. Sorry, I tried to use macOS recent weeks, and I don't have any thoughts about it.

Reply:
Interestingly I just found the `--universal` flag, maybe it is possible. That would be a game changer if we could cross compile for aarch64.

Reply:
I will leave this for a separate PR. The main reason for this CL is so that the Linux native libraries are built using an older Ubuntu.